CHANGELOG.md

```diff
@@ -1,3 +1,514 @@
+
+Read more about the toolbox pod here: https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/rook-ceph-storage.md#enable-and-access-toolbox.
+
+> **NOTE**: If you see this error `[errno 5] RADOS I/O error (error connecting to the cluster)` in toolbox pod then tag the toolbox pod image to a specific version using this command: `kubectl -n rook set image deploy rook-ceph-tools rook-ceph-tools=rook/ceph:v1.3.2`.
+
+- Ceph Status
+
+Run the following in the toolbox pod:
+
+```
+watch ceph status
+```
+
+Ensure that the output says that the health is `HEALTH_OK`. Match the output such that everything looks fine as explained here: https://rook.io/docs/rook/master/ceph-upgrade.html#status-output.
+
+
+- Pods in rook namespace:
+
+Watch the pods status in another from the `rook` namespace in another terminal window. Just running this will be enough:
+
+```
+watch kubectl -n rook get pods -o wide
+```
+
+- Watch for the rook version update
+
+Run the following command to keep an eye on the rook version update as it is rolls down for all the components:
+
+```
+watch --exec kubectl -n rook get deployments -l rook_cluster=rook -o jsonpath='{range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \trook-version="}{.metadata.labels.rook-version}{"\n"}{end}'
+```
+
+You should see that `rook-version` slowly changes to `v1.4.2`.
+
+- Watch for the Ceph version update
+
+Run the following command to keep an eye on the Ceph version update as the new pods come up:
+
+```
+watch --exec kubectl -n rook get deployments -l rook_cluster=rook -o jsonpath='{range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \tceph-version="}{.metadata.labels.ceph-version}{"\n"}{end}'
+```
+
+TODO:
+You should see that `ceph-version` slowly changes to `15.`
+
+- Keep an eye on the events in the rook namespace
+
+```
+kubectl -n rook get events -w
+```
+
+- Ceph Dashboard
+
+Keep it open in one window, but sometimes it is more hassle than any help. It keeps reloading and logs you out automatically. See this on how to access the dashboard: https://github.com/kinvolk/lokomotive/blob/master/docs/how-to-guides/rook-ceph-storage.md#access-the-ceph-dashboard.
+
+- Grafana dashboards
+
+Keep an eye on the Grafana dashboard but the data here will always be old and the most reliable state of the system will come from the watch running inside toolbox pod.
+
+- Run updates
+
+```
+kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/master/assets/charts/components/rook/templates/resources.yaml
```

Comment:
Perhaps link this to a specific commit instead of the master branch to make sure it's future proof?

Reply:
Ideally to tag, good point :)